warehouse/query-core/src/main/java/datawave/query/iterator/QueryOptions.java

```diff
@@ -1672,6 +1684,10 @@ public static String compressOption(final String data, final Charset characterSe
         return new String(Base64.encodeBase64(byteStream.toByteArray()));
     }
     
+    /*
+     * TODO: The following two methods are more appropriately generified and put into utils.There is nothing indexOnly field specific about them at all. They
```

Comment:
then just rename the method (e.g. buildStringFromFieldList)

Reply:
I confirmed that PF does not use this method

Reply:
done in next commit